src/Elastic.Apm.Azure.Functions/AzureFunctionsExtensions.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to Elasticsearch B.V under one or more agreements.
+// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information
+
+using System.Collections.Generic;
+using System.Linq;
+using Elastic.Apm.DiagnosticSource;
+using Microsoft.Azure.Functions.Extensions.DependencyInjection;
+
+namespace Elastic.Apm.Azure.Functions
+{
+	public static class AzureFunctionsExtensions
+	{
+		public static IFunctionsHostBuilder AddElasticApm(this IFunctionsHostBuilder builder)
```

Comment:
- [ ] This definitely needs another iteration:
  * documentation
  * additional arguments (configuration, subscribers, logger, ...)